src/sinks/redis.rs

```diff
@@ -342,12 +342,12 @@ impl Service<Vec<RedisKvEntry>> for RedisSink {
     type Error = RedisError;
     type Future = BoxFuture<'static, RedisPipeResult>;
 
-    // Emission of Error internal event is handled upstream by the caller
+    // Emission of an internal event in case of errors is handled upstream by the caller.
```

Comment:
:+1: yeah there was a PR a few weeks ago that I had where we discussed it and now that I see it above, I liked that better.
I think the initial concern was I had it as a doc comment for the function and that was bad because it overrides the display of the trait function doc comment, so as long as its a standard `//` comment like you have I think above is :ok_hand: